OWNERS.md

```diff
@@ -51,3 +51,5 @@ matter expert reviews. Feel free to loop them in as needed.
   * Bazel/build.
 * Joshua Marantz ([jmarantz](https://github.com/jmarantz)) (jmarantz@google.com)
   * abseil and performance work.
+* Wayne Zhang ([qiwzhang](https://github.com/qiwzhang)) (qiwzhang@google.com)
```

Comment:
I meant [CODEOWNERS](https://github.com/envoyproxy/envoy/blob/master/CODEOWNERS), add me and you there.

btw, welcome to be friends :)

Reply:
Done. Added line in CODEOWNERS.   Removed myself from OWNERS.md